pages/data-design/pinned-properties.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: 'guide'
+title: 'Pinned properties'
+---
+
+import Link from '../../components/Link';
+
+**Pinned Properties** is a way to make specific property be sent with the same constant value all the time.
+
+## How to use property pinning
+
+Use property pinning to assign constant value to a property. When the value of a property is pinned it will always be sent as the pinned value. 
```

Comment:
```suggestion
Use pinned properties to assign constant value to a property. When the value of a property is pinned it will always be sent as the pinned value. 
```